engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -151,12 +151,12 @@ class MetadataBuilderActorSpec extends TestKit(ActorSystem("Metadata"))
 
   it should "build lexigraphically sorted JSON list structure from dotted key syntax" in {
     val eventBuilderList = List(
-      ("l1[2]", "l12", currentTime),
-      ("l1[k]", "l1k", currentTime),
-      ("l1[3]", "l13", currentTime),
-      ("l1[i]", "l1i", currentTime),
-      ("l1[10]", "l110", currentTime),
-      ("l1[j]", "l1j", currentTime)
+      ("l1[2]", "l12", OffsetDateTime.now),
+      ("l1[k]", "l1k", OffsetDateTime.now),
+      ("l1[3]", "l13", OffsetDateTime.now),
+      ("l1[i]", "l1i", OffsetDateTime.now),
+      ("l1[10]", "l110", OffsetDateTime.now),
+      ("l1[j]", "l1j", OffsetDateTime.now)
```

Comment:
Is there a danger that `OffsetDateTime.now` will be slightly different for each of these lines (since they take a small amount of time to run)?